--- mldonkey-2.9.0/config/configure.in	2007-09-09 00:13:01.000000000 +0000
+++ mldonkey-2.9.0/config/configure.in	2007-09-09 00:19:21.000000000 +0000
@@ -486,7 +486,7 @@
 else
   OCAMLVERSION=`$OCAMLC -v | sed -n -e 's|.*version* *\(.*\)$|\1|p' `
   case "$OCAMLVERSION" in
-    "$REQUIRED_OCAML"*|3.09.*|3.08.4*|3.08.3*) ;;
+    "$REQUIRED_OCAML"*|3.10.*|3.09.*|3.08.4*|3.08.3*) ;;
     *)
         echo "Need build"
         BUILD_OCAML=yes
